eng/Versions.props

```diff
@@ -30,6 +30,7 @@
     <!-- Packages from dotnet/runtime -->
     <MicrosoftBclTimeProviderVersion>9.0.0-rc.1.24419.2</MicrosoftBclTimeProviderVersion>
     <MicrosoftExtensionsCachingAbstractionsVersion>9.0.0-rc.1.24419.2</MicrosoftExtensionsCachingAbstractionsVersion>
+    <MicrosoftExtensionsCachingMemoryVersion>9.0.0-rc.1.24419.2</MicrosoftExtensionsCachingMemoryVersion>
```

Comment:
Can we add these to Version.Details.xml as well in order to onboard into dependency flow so that these versions being added here are automatically updated each time there is a new aspnetcore build? Same goes for the new property bellow. If you have questions on how to do it I'm happy to help.